src/Tizen.Sensor/Tizen.Sensor/BatchData/HeartRateMonitorLEDGreenBatchData.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Tizen.Sensor.BatchData
+{
+    /// <summary>
+    /// The HeartRateMonitorLEDGreenBatchData class is used for storing value of HeartRateMonitorLEDGreenBatch sensor.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class HeartRateMonitorLEDGreenBatchData : BatchData
+    {
+        /// <summary>
+        /// Initializes a new instance of the HeartRateMonitorLEDGreenBatchData class.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <param name="timeStamp">Timestamp of HeartRateMonitorLEDGreenBatchData sensor event.</param>
+        /// <param name="accuracy">Sensor accuracy of HeartRateMonitorLEDGreenBatchData sensor event.</param>
+        /// <param name="green">green of the HeartRateMonitorLEDGreenBatchData sensor event.</param>
+        /// <param name="accel_x">accel_x of HeartRateMonitorLEDGreenBatchData sensor event.</param>
+        /// <param name="accel_y">accel_y of HeartRateMonitorLEDGreenBatchData sensor event.</param>
+        /// <param name="accel_z">accel_z of HeartRateMonitorLEDGreenBatchData sensor event.</param>
+        /// <param name="index">rri of HeartRateMonitorLEDGreenBatchData sensor event.</param>
+        public HeartRateMonitorLEDGreenBatchData(TimeSpan timeStamp, SensorDataAccuracy accuracy, uint green, uint accel_x, uint accel_y, uint accel_z, uint index) : base(timeStamp, accuracy)
+        {
+            Green = green;
+            ACCEL_X = accel_x;
+            ACCEL_Y = accel_y;
+            ACCEL_Z = accel_z;
+            Index = index;
+        }
+
+        /// <summary>
+        /// Gets the value of the Green LED ADC Value.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <value> Green </value>
+        public uint Green { get; }
+
+        /// <summary>
+        /// Gets the X axis's uncalibrated value of the gravitational acceleration.
+        /// </summary>
+        /// <since_tizen> 8 </since_tizen>
+        /// <value> ACCEL_X </value>
+        public uint ACCEL_X { get; }
```

Comment:
✔️ DO use PascalCasing for all public member, type, and namespace names consisting of multiple words.
```suggestion
        public uint AccelerationX { get; }
```